### ISSUE TITLE
Cannot see dashboard configuration with admin account

### ISSUE THREAD

**Issue by halduaij** [heavyai/heavydb#685]
Hello,

I followed the instructions here https://docs.omnisci.com/immerse/working-with-dashboards/configuring-a-dashboard , but I cannot see the configuration icon, and it was not disabled. Can someone help me? as I need to configure the dashboard to change the axis label fonts and such.
Thank you.

[
![missing_icon](https://user-images.githubusercontent.com/92260325/136699086-9f80346e-c861-492a-8a0d-efad6c453956.png)

**Comment by cdessanti** [comment]
Hi,

I'm using the admin user on Immerse with free edition and I haven't any problem

![Screenshot from 2021-10-10 21-24-45](https://user-images.githubusercontent.com/30822551/136710398-d08afc38-cdeb-4149-a554-d3d0c38da410.png)

**Comment by halduaij** [comment]
What app are you using to run omnisci?

**Comment by halduaij** [comment]
How can I see server.json file?

**Comment by halduaij** [comment]
![image](https://user-images.githubusercontent.com/92260325/136814731-714cfd09-fd72-4396-8079-46cf311e11f2.png)

**Comment by cdessanti** [comment]
well, the first login is ad admin. If you log out manually and then login with the user test/test, you shouldn't see the icon (check if it's the same one you are seeing in your university system)

to logout manually use the icon located into the right upper corner
![image](https://user-images.githubusercontent.com/30822551/136972638-770ee9b9-aa66-4935-8991-34e21ae5f780.png)

**Comment by cdessanti** [comment]
Yes, so you browser and your network looks ok; peraphs you should ask to verify the configuration of the omnisci in your network.

Using the test user are seeing exactly what you seen when you connect to your DB?

**Comment by halduaij** [comment]
When I try to connect as test user in my db it says invalid credentials.

**Comment by cdessanti** [comment]
Because I created and ad-hoc user that I called test on my system, for testing purposes.
The only use that is automatically created when a system is initialized is admin.

If you want to create a user called test, connect to immerse with an admin account,  go in the SQL editor and type

Create user test (password='test');

Then run another query to give the permission on a specific database (in the following example I am using the omnisci database)

Grant all on database omnisci to test;

Running those commands you will be sure that your user is an admin user.

Candido

Scarica Outlook per Android<https://aka.ms/AAb9ysg>

________________________________
Da: halduaij ***@***.***>
Inviato: martedì 12 ottobre 2021, 23:58
A: omnisci/omniscidb
Cc: Candido Dessanti; Comment
Oggetto: Re: [omnisci/omniscidb] Cannot see dashboard configuration with admin account (#685)


When I try to connect as a test user in my DB it says invalid credentials.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fomnisci%2Fomniscidb%2Fissues%2F685%23issuecomment-941628987&data=04%7C01%7C%7C77a0babbacd64eefd5fa08d98dcb64c5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637696726967649045%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=pXBvn2BY8P9UEOUGXM6FBe%2FVgfzcHC9SEtIMB94dSJg%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAHLFBFYAJLNUHD4BO6AO4A3UGSVPNANCNFSM5FWQKDDA&data=04%7C01%7C%7C77a0babbacd64eefd5fa08d98dcb64c5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637696726967659039%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=dePBx7p36mu0V6cfkQqzcq3GwxMmKmEihRr88rX9Uso%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C77a0babbacd64eefd5fa08d98dcb64c5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637696726967659039%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=7KXmzF96%2BzOlRiI%2Fh5rzFS02i2QnoDPqflrIZrLe9sE%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C77a0babbacd64eefd5fa08d98dcb64c5%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637696726967669039%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=RB0e3vx9oUvi%2BVAI0PGNV06mnLZH%2B9zljNYO8ivs8sc%3D&reserved=0>.

**Comment by psilvest** [comment]
This issue has been inactive for a while, and the project has changed substantially. If this still occurs on the latest release, please add reproduction steps and environment details.
